feature-group-definitions/calc-constants.dist.yml

```diff
@@ -0,0 +1,19 @@
+# Generated from: calc-constants.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: calc() constants
+description: The `e`, `pi`, `infinity`, and `NaN` constants are accepted in CSS math functions such as `calc()`.
+spec: https://drafts.csswg.org/css-values-4/#calc-keywords
+status:
+  baseline: false
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/CSS/calc-constant#browser_compatibility this is probably correct, but I can't find a Chrome bug for the missing bits, or any tests in WPT. I will back it out to consider separately.

Reply:
It was wrong: https://github.com/mdn/browser-compat-data/pull/22847